console/client/.eslintrc.cjs

```diff
@@ -24,6 +24,7 @@ module.exports = {
   rules: {
     'react/react-in-jsx-scope': 'off',
     'react/jsx-uses-react': 'off',
+    'react/prop-types': 'off',
```

Comment:
Do we want to disable this rule? How does type checking work with this off?

Reply:
Yeah we do if we're using typescript as it's actually not necessary to lint for this.  Further it also allows to use the chained component sub-component pattern.